packaging/linux/run_keybase

```diff
@@ -161,7 +161,9 @@ main() {
 
   echo 'Success!'
   # Magical squirrel produced by https://github.com/erkin/ponysay
-  cat /opt/keybase/crypto_squirrel.txt
+  if [ $LC_CTYPE != "C"]; then
```

Comment:
Make sure to quote this variable expansion.

Reply:
This patch will help, but not if there is no `LC_CTYPE`.  You should check `LC_ALL` and `LANG` as well, and if none of those are set.  Linux also has a command-line utility locale that will print these out, so something like `locale | grep '^LC_CTYPE` might help.

Reply:
But more importantly, even in UTF-8, not everyone wants to lose a page of terminal history.  Can you suppress the squirrel if a file `~/.config/keybase/hush` exists?